src/mono/wasm/wasm.proj

```diff
@@ -409,6 +409,8 @@
       <_CmakeEnvironmentVariable Include="WASM_ENABLE_SIMD=0" Condition="'$(WasmEnableSIMD)' == 'false'" />
```

Comment:
@radical @radekdoulik Why do we have these conditions here? We never build runtime with these settings, aren't we?

Reply:
Could be my mistake ...